src/main/java/org/jitsi/jicofo/JitsiMeetConferenceImpl.java

```diff
@@ -2620,7 +2620,27 @@ private void reInviteParticipants(List<Participant> participants)
         {
             for (Participant participant : participants)
             {
+                BridgeSession session = participant.getBridgeSession();
```

Comment:
Given a meeting with bridges A, B, and C with a participant 1 on bridge A: if participant 1 moves from bridge A to bridge B, would bridge C have previously seen no change but will now see participant 1 removed as an Octo participant and then re-added as an Octo participant?

Reply:
Yes, as far as I can see this is exactly right.